pom.client.xml

```diff
@@ -1203,5 +1203,6 @@
     <module>sdk/storage/azure-storage-file-share</module>
     <module>sdk/storage/azure-storage-file-datalake</module>
     <module>sdk/storage/azure-storage-queue</module>
+    <module>sdk/textanalytics/azure-ai-textanalytics</module>
```

Comment:
Along with the module reference you're going to also want to add the group/ditle/package configuration for the javadoc plugin. It's the first plugin under the <pluginManagement> which starts around line 383.

Reply:
Thanks for pointing it out